example/cumulus-tf/variables.tf

```diff
@@ -116,6 +116,12 @@ variable "prefix" {
   type = string
 }
 
```

Comment:
The added variables are not needed

Reply:
All of them ?

Reply:
yes.  We are not providing them in terraform.tfvars file